packages/jest-runtime/src/index.js

```diff
@@ -432,17 +394,8 @@ class Runtime {
     localModule.paths = this._resolver.getModulePaths(dirname);
     localModule.require = this._createRequireImplementation(filename, options);
 
-    const script = transform(
-      filename,
-      options && options.isInternalModule ? null : this._config,
-      shouldCollectCoverage ? {
-        instrument: source => collectors[filename].getInstrumentedSource(
-          source,
-          filename,
-          '$JEST',
-        ),
-      } : null,
-    );
+    const script = transform(filename, this._config, {isInternalModule});
```

Comment:
YESSS. I want to say for the record that the instrument option was always meant to be temporary so that I could push the `vm.Script` creation into `transform` and I only did it because I knew you made these refactors.